editions/dev/tiddlers/ldorigo/High-Level Overview/Hooks/Hook_ th-page-refreshing.tid

```diff
@@ -0,0 +1,15 @@
+created: 20190111150102847
+modified: 20190111150102847
+tags: HookMechanism
+title: Hook: th-page-refreshing
+type: text/vnd.tiddlywiki
+
+This hook notifies plugins that a page refresh is just about to occur. It is typically used to apply pre-rendering effects.
+
+Hook function parameters:
+
+* (none)
+
+Return value:
+
+* (none)
```

Comment:
TODO: It would be nice, if the hooks could link to "real world" examples, if we can find some ;)

Reply:
Yes, very much so. I only aggregated/cleaned existing documentation so far, there is still a huge amount of work to be done to actually make it complete.